src/components/Wizard/CreateVmWizard/CreateVmWizard.js

```diff
@@ -329,6 +330,7 @@ CreateVmWizard.propTypes = {
   k8sPatch: PropTypes.func.isRequired,
   k8sKill: PropTypes.func.isRequired,
   onHide: PropTypes.func.isRequired,
+  templatesLoaded: PropTypes.bool,
```

Comment:
I left comment regarding this prop in https://github.com/openshift/console/pull/2157

Reply:
Agreed in 2157